image/v3/modifying/drawing.md

```diff
@@ -190,8 +190,8 @@ $image = $manager->read('test.png');
 $image->drawLine(function ($line) {
     $line->from(10, 10); // starting point of line
     $line->to(300, 100); // ending point
-    $line->color('ff00ff'); // color of line
-    $line->width(5); // line width in pixels
+    $line->setBackgroundColor('ff00ff'); // color of line
+    $line->setWidth(5); // line width in pixels
```

Comment:
Please revert. These were correct calls of class `Intervention\Image\Geometry\Factories\LineFactory::class`.